addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -113,6 +113,8 @@ class DescriptionScreen extends Component {
         this.industrySelection = useRef('industrySelection');
         this.state = useStore();
         this.orm = useService('orm');
+        // TODO: if we put directly in the xml then export POT file, this won't appear at all, probably a bug
+        this.state.industryPlaceholder = _t("Type an industry (Fashion, Electronic store, etc...)")
```

Comment:
What you should do then instead of moving it to js, is keep it in XML but in a `t-set` variable, and then use that variable in the placeholder value.
Moving the string in its own `t-set` will allow the term to be exported.

```xml
<t t-set="var_name">Some text to be translated</t>
```

Reply:
@rdeodoo Done sir, i add a small comment for other people if they ever question why i use `t-set`